packages/maskbook/src/plugins/FileService/define.tsx

```diff
@@ -44,6 +44,6 @@ export const FileServicePluginDefine: PluginConfig = {
             },
         ],
     ]),
-    pageInspector: FileServiceCompositionUI,
+    PageComponent: FileServiceCompositionUI,
```

Comment:
https://github.com/DimensionDev/Maskbook/commit/2530b062b29708254c41ec4d458eb42bd28f6d1c#diff-2e0e90c36cac04931dfea7ddb0d1ecb0a25915cf093447cd91283aabc4658063L15

`pageInspector` has been renamed to `PageComponent`